server/helpers/identification.js

```diff
@@ -0,0 +1,24 @@
+import { UserRole, User, Role } from '../database/models';
+
+const findIdByEmail = email => User.findOne({
```

Comment:
This function should take in response as an argument and use it to return proper error messages.

Reply:
> This function should take in response as an argument and use it to return proper error messages.

Done